KakapoExample/KakapoTests/checkUrlTest.swift

```diff
@@ -15,24 +15,26 @@ class checkUrlTest: QuickSpec {
     override func spec() {
         describe("#checkUrl") {
             it("should return nil if the requested url doesn't match the declared one") {
-                expect(parseUrl("/users/:id", requestURL: "/comments/1")).to(beNil())
```

Comment:
This file should be called `DecomposeURLTests`? describe could be more describing and these tests are actually testing multiple behavior in single tests (we may rewrite it as part of another PR if you prefer but create an issue then)